src/common/commonutils/PackageUtils.c

```diff
@@ -187,6 +209,7 @@ int InstallOrUpdatePackage(const char* packageName, void* log)
     
     if (g_aptGetIsPresent)
     {
+        AptGetUpdateOnce(log);
```

Comment:
>AptGetUpdateOnce(log); [](http://example.com/codeflow?start=8&length=22)

How long can this take? There is a concern of ending in Azure Policy timeout on slow machines due to lengthy package operations during audit